src/appendices/spring/thymeleaf.rst

```diff
@@ -0,0 +1,108 @@
+.. _thymeleaf-commands:
+
+Thymeleaf Commands
+==================
+
+Thymeleaf is a template engine and Thymeleaf templates make up the views of our MVC application.
+In order to create our templates, we use attributes to pass data through controllers to our views out of the model.
+
+Attributes
+----------
+
+Thymeleaf has many attributes that we could use. In this book, we focus on a few key ones.
+
+Displaying Data
+^^^^^^^^^^^^^^^
+
+``th:text`` dynamically populates the contents of an HTML element.
+
+.. sourcecode:: html
+
+   <p th:text = "Hello World!">text</p>
```

Comment:
Is the "text" between the ``<p>`` tags necessary, since we can display text without it (e.g. ``<h1 th:text="${title}"></h1>``)?  This is especially true with ``<p th:text = "Hello World!">text</p>``, since the text we want to display is hard-coded inside the tag.

If we want it to represent a default option, then we should be specific in the syntax.  ``<p th:text = "${hello}">Default Text</p>``.

I'd also add a quick comment or two about when the default text would be displayed.

Reply:
Nope! I was just not paying attention when I was reading some examples. Thanks!